README.md

```diff
@@ -62,15 +62,16 @@ $pledge = null;
  * Given a resource in the document, you can access its relationships with ->relationship->($relationship_name)
  * or check for a relationship's existence with ->has('relationships.relationship_name')
  */
-if ($user->has('relationships.pledges')) {
+if ($patron->has('relationships.pledges')) {
     /*
      * To look up the full resource that the relationship is referencing,
-     * we have extended [art4/json-api-client/ResourceIdentifier](https://github.com/Art4/json-api-client/blob/master/docs/objects-resource-identifier.md)
+     * we have extended art4/json-api-client/ResourceIdentifier
```

Comment:
Links are formatted inside code blocks, so this was hard to read.

Reply:
👍